src/OrleansSQLUtils/Storage/Provider/AdoGrainKey.cs

```diff
@@ -73,17 +74,18 @@ public byte[] GetHashBytes()
                 bytes = ToGuidKey(N0Key, N1Key).ToByteArray();
             }
 
+            var utf8 = new UTF8Encoding();
```

Comment:
@attilah Do you mean this in particular or all the othes too?

It appears not be thread safe. If I use the static property, there will incorrect results in high-concurrency scenario. I newed up them all just be on the robust side. Should gargage or performance become a problem, there looks like be more fruitful considerations too.